clusterman/migration/worker.py

```diff
@@ -131,4 +157,44 @@ def event_migration_worker(migration_event: MigrationEvent, worker_setup: Worker
     :param MigrationEvent migration_event: event instance
     :param WorkerSetup worker_setup: migration setup
     """
-    pass  # TODO
+    prescaling_applied = False
+    manager = PoolManager(migration_event.cluster, migration_event.pool, SUPPORTED_POOL_SCHEDULER, fetch_state=False)
+    connector = cast(KubernetesClusterConnector, manager.cluster_connector)
+    connector.set_label_selectors(migration_event.label_selectors, add_to_existing=True)
+    manager.reload_state()
+    try:
+        if worker_setup.disable_autoscaling:
+            logger.info(f"Disabling autoscaling for {migration_event.cluster}:{migration_event.pool}")
+            disable_autoscaling(
+                migration_event.cluster,
+                migration_event.pool,
+                SUPPORTED_POOL_SCHEDULER,
+                time.time() + worker_setup.expected_duration,
+            )
+        if worker_setup.prescaling:
+            nodes = manager.get_node_metadatas(AWS_RUNNING_STATES)
+            offset = worker_setup.prescaling.of(len(nodes))
```

Comment:
just curious: how does this work? i don't see an `of` method in [WorkerSetup](https://github.com/Yelp/clusterman/blob/4fedeb246c182310ae2598335b46b7f11754109f/clusterman/migration/settings.py#L46:7)

Reply:
The `prescaling` property is an instance of [`PoolPortion`](https://github.com/Yelp/clusterman/blob/master/clusterman/migration/settings.py#L46). Just an abstanction to transparently support both percentages and absolute values in the configuration file.

(yes, sometimes I definitely go for a bit too much OOP - and now the whole Java community will be angry with me for dynamically setting a method instead of using a factory :P)